### DISCUSSION TITLE
inav failsave symbol red - what does it mean

### DISCUSSION THREAD

**Discussion by ghost** [iNavFlight/inav-configurator#1694] [General]
The INAV 5.1 failsave symbol stays red, dont know why?

Could you give me some hints please?

*

*PLEASE MAKE SURE YOU READ AND UNDERSTAND THE SOCIAL MEDIA SUPPORT CHANNELS. QUESTIONS ABOUT FLASHING, CONFIGURING, PILOTING MAY BE CLOSED WITHOUT FURTHER INTERACTION.**

* [INAV Discord Server](https://discord.gg/peg2hhbYwN)
* [INAV Official on Facebook](https://www.facebook.com/groups/INAVOfficial)
* [INAV Official on Telegram](https://t.me/INAVFlight)

**Please double-check that nobody reported the issue before by using search in this bug tracker.**

**PLEASE DELETE THE TEXT ABOVE AFTER READING AND UNDERSTANDING IT**

****

## Steps to Reproduce

1.
2.
3.
4.

## Additional context

---

- FC Board name and vendor:
- INAV version string:

**Comment by MrD-RC** [comment]
It means you do not have a valid RF connection to your transmitter. It is in failsafe.

**Reply by ghost** [reply]
But RF connection works besides of  throttle. Even this could be seen at the INAV configurator, but the motor itself does not react with rc transmitter. All other servos do

**Reply by MrD-RC** [reply]
If the parachute is red, your receiver is sending a failsafe bit. 

Maybe you should describe your setup. INAV believes that there is a failsafe happening.

**Reply by ghost** [reply]
Simple setup: Matek F405 WMN, INAV 5.1, Jeti with SBus Protocol (JETIEXBUS Protocol ist NOT working, maybe an Error?)
Beitian GPS 220
All is working correctly besides the throttle channel. 
All servos are working at INAV screen and in reality, besides throttle. This is not working up to now
I tried several protocolls, but only SBus is working.
And having the failsafe symbol in red on INAV screen
